docs/aerodromes/classc/Sydney.md

```diff
@@ -296,6 +296,8 @@ The Operational Information field should be updated based on the runway mode and
 | `EXP INSTR APCH`, or<br>`EXP INSTR APCH THEN INDEP VISUAL APCH WHEN VISUAL` | `INDEP PARL DEPS IN PROG` |
 | `EXP INDEP VISUAL APCH` | `INDEP PARL APPROACHES AND DEPS IN PROG` |
 | SODROPS | `SIMUL OPP DIR PARL RWY OPS IN PROG` |
+| Curfew Mode Operations | AEST: `CURFEW RWY NOMINATION. CURFEW IN OPERATION UNTIL TIME 2000.` |
+| | AEDT: `CURFEW RWY NOMINATION. CURFEW IN OPERATION UNTIL TIME 1900.` |
```

Comment:
```suggestion
| Curfew Mode Operations | AEST: `CURFEW RWY NOMINATION. CURFEW IN OPERATION UNTIL TIME 2000.` <br>AEDT: `CURFEW RWY NOMINATION. CURFEW IN OPERATION UNTIL TIME 1900.` |
```

Having these both in a single cell on new lines might help to correlate them with the title in column 1 better